core/winnt/src/TWinNTSystem.cxx

```diff
@@ -1762,9 +1762,17 @@ void TWinNTSystem::DispatchSignals(ESignals sig)
       fSigcnt++;
    }
    else {
-      StackTrace();
-      if (TROOT::Initialized()) {
-         ::Throw(sig);
+      if (gExceptionHandler) {
+         //sig is ESignal, should it be mapped to the correct signal number?
+         if (sig == kSigFloatingException) _fpreset();
+         gExceptionHandler->HandleException(sig);
+      } else {
+         //map to the real signal code + set the
+         //high order bit to indicate a signal (?)
+         StackTrace();
+         if (TROOT::Initialized()) {
+             ::Throw(sig);
+         }
```

Comment:
To illustrate the different behaviors, here is the original stack trace (without those changes):
```
==========================================
=============== STACKTRACE ===============
==========================================


================ Thread 0 ================
  libCling!llvm::TargetLowering::SimplifyDemandedBits()
  libCling!llvm::TargetLowering::DAGCombinerInfo::CommitTargetLoweringOpt()
  libCling!llvm::TargetLowering::DAGCombinerInfo::CommitTargetLoweringOpt()
  libCling!llvm::SDNode::use_size()
  libCling!llvm::SDNode::use_size()
  libCling!llvm::EVT::changeVectorElementTypeToInteger()
  libCling!llvm::TargetLowering::DAGCombinerInfo::CommitTargetLoweringOpt()
  libCling!llvm::SelectionDAG::Combine()
  libCling!llvm::SelectionDAGISel::CodeGenAndEmitDAG()
  libCling!llvm::SelectionDAGISel::SelectBasicBlock()
  libCling!llvm::SelectionDAGISel::SelectAllBasicBlocks()
  libCling!llvm::SelectionDAGISel::runOnMachineFunction()
  libCling!llvm::SmallVectorImpl<std::pair<llvm::SDValue,llvm::SDNode *> >::resize()
  libCling!llvm::MachineFunctionPass::runOnFunction()
  libCling!llvm::FPPassManager::runOnFunction()
  libCling!llvm::FPPassManager::runOnModule()
  libCling!llvm::FPPassManager::runOnModule()
  libCling!llvm::legacy::PassManagerImpl::run()
  libCling!llvm::orc::SimpleCompiler::operator()()
  libCling!llvm::orc::IRCompileLayer<cling::IncrementalJIT::RemovableObjectLinkingLayer,llvm::orc::SimpleCompiler>::addModule()
  libCling!<lambda_65945f5355df0d8633fb0961423f4ff4>::operator()()
  libCling!?_Do_call@?$_Func_impl_no_alloc@V<lambda_65945f5355df0d8633fb0961423f4ff4>@@V?$Expected@_K@llvm@@$$V@std@@EAE?AV?$Expected@_K@llvm@@XZ()
  libCling!llvm::JITSymbol::getAddress()
  libCling!cling::IncrementalJIT::getSymbolAddress()
  libCling!cling::IncrementalExecutor::getPointerToGlobalFromJIT()
  libCling!cling::Interpreter::compileFunction()
  C:\Users\bellenot\github\etejedor\core\metacling\src\TClingCallFunc.cxx(271): TClingCallFunc::compile_wrapper()
  C:\Users\bellenot\github\etejedor\core\metacling\src\TClingCallFunc.cxx(1096): TClingCallFunc::make_wrapper()
  C:\Users\bellenot\github\etejedor\core\metacling\src\TClingCallFunc.cxx(2233): TClingCallFunc::IFacePtr()
  C:\Users\bellenot\github\etejedor\core\metacling\src\TCling.cxx(7755): TCling::CallFunc_IFacePtr()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\cppyy-backend\clingwrapper\src\clingwrapper.cxx(583): GetCallFunc()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\cppyy-backend\clingwrapper\src\clingwrapper.cxx(623): WrapperCall()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\cppyy-backend\clingwrapper\src\clingwrapper.cxx(725): Cppyy::CallConstructor()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\Executors.cxx(98): GILCallConstructor()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\Executors.cxx(659): CPyCppyy::`anonymous namespace'::ConstructorExecutor::Execute()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\CPPMethod.cxx(62): CPyCppyy::CPPMethod::CallFast()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\CPPMethod.cxx(127): CPyCppyy::CPPMethod::CallSafe()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\CPPMethod.cxx(588): CPyCppyy::CPPMethod::Execute()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\CPPConstructor.cxx(85): CPyCppyy::CPPConstructor::Call()
  C:\Users\bellenot\github\etejedor\bindings\pyroot_experimental\cppyy\CPyCppyy\src\CPPOverload.cxx(629): CPyCppyy::`anonymous namespace'::mp_call()
  python37!PyObject_Call()
  python37!PyType_Ready()
  python37!PyType_Name()
  python37!PyObject_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyObject_Call()
  python37!PyFunction_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyObject_Call()
  python37!PyFunction_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyMethod_New()
  python37!PyObject_Call()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyType_Ready()
  python37!PyObject_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyMethod_New()
  python37!PyObject_Call()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyType_Ready()
  python37!PyObject_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyMethod_New()
  python37!PyObject_Call()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyType_Ready()
  python37!PyObject_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyMethod_New()
  python37!PyObject_Call()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyType_Ready()
  python37!PyObject_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyObject_Call()
  python37!PyFunction_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyObject_Call()
  python37!PyFunction_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_FastCallDict()
  python37!PyObject_Call_Prepend()
  python37!PyType_Ready()
  python37!PyType_Name()
  python37!PyObject_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyEval_EvalCode()
  python37!PyAST_Optimize()
  python37!PyAST_Optimize()
  python37!PyMethodDef_RawFastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallKeywords()
  python37!PyEval_GetFuncDesc()
  python37!PyEval_EvalFrameDefault()
  python37!PyEval_EvalCodeWithName()
  python37!PyFunction_FastCallDict()
  python37!PyObject_Call()
  python37!Py_hashtable_copy()
  python37!PyMainInterpreterConfig_Read()
  python37!PyMainInterpreterConfig_Read()
  python37!Py_Main()
  python!??
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 1 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 2 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 3 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 4 ================
  ntdll!ZwDelayExecution()
  KERNELBASE!Sleep()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(4351): TWinNTSystem::TimerThread()
  C:\Users\bellenot\github\etejedor\core\winnt\inc\TWinNTSystem.h(99): TWinNTSystem::ThreadStub()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 5 ================
  win32u!NtUserGetMessage()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(494): `anonymous namespace'::GUIThreadMessageProcessingLoop()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 6 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 7 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 8 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 9 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 10 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 11 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 12 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

==========================================
============= END STACKTRACE =============
==========================================
```

And here the one with those changes:
```
==========================================
=============== STACKTRACE ===============
==========================================


================ Thread 0 ================
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(450): `anonymous namespace'::SigHandler()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(391): `anonymous namespace'::sighandler()
  ucrtbase!seh_filter_exe()
  python!??
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 1 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 2 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 3 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 4 ================
  ntdll!ZwDelayExecution()
  KERNELBASE!Sleep()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(4349): TWinNTSystem::TimerThread()
  C:\Users\bellenot\github\etejedor\core\winnt\inc\TWinNTSystem.h(99): TWinNTSystem::ThreadStub()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 5 ================
  win32u!NtUserGetMessage()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(494): `anonymous namespace'::GUIThreadMessageProcessingLoop()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 6 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 7 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 8 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 9 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 10 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 11 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 12 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

==========================================
============= END STACKTRACE =============
==========================================


==========================================
=============== STACKTRACE ===============
==========================================


================ Thread 0 ================
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(450): `anonymous namespace'::SigHandler()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(391): `anonymous namespace'::sighandler()
  ucrtbase!raise()
  ucrtbase!abort()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(3909): TWinNTSystem::Abort()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(1781): TWinNTSystem::DispatchSignals()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(450): `anonymous namespace'::SigHandler()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(391): `anonymous namespace'::sighandler()
  ucrtbase!seh_filter_exe()
  python!??
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 1 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 2 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 3 ================
  ntdll!ZwWaitForWorkViaWorkerFactory()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 4 ================
  ntdll!ZwDelayExecution()
  KERNELBASE!Sleep()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(4349): TWinNTSystem::TimerThread()
  C:\Users\bellenot\github\etejedor\core\winnt\inc\TWinNTSystem.h(99): TWinNTSystem::ThreadStub()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 5 ================
  win32u!NtUserGetMessage()
  C:\Users\bellenot\github\etejedor\core\winnt\src\TWinNTSystem.cxx(494): `anonymous namespace'::GUIThreadMessageProcessingLoop()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 6 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 7 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 8 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 9 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 10 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 11 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

================ Thread 12 ================
  ntdll!ZwWaitForMultipleObjects()
  KERNELBASE!WaitForMultipleObjects()
  libopenblas.JKAMQ5EVHIVCPXP2XZJ!openblas_read_env()
  KERNEL32!BaseThreadInitThunk()
  ntdll!RtlGetAppContainerNamedObjectPath()
  ntdll!RtlGetAppContainerNamedObjectPath()

==========================================
============= END STACKTRACE =============
==========================================
```